tbx/project_styleguide/templates/patterns/styleguide/components/components.html

```diff
@@ -0,0 +1,7 @@
+<div class="p-4">
+    <p class="underline mb-2">Avatar</p>
+    {% include "patterns/atoms/avatar/avatar.html" %}
+
+    <p class="underline mt-8 mb-2">Avatar (small)</p>
+    {% include "patterns/atoms/avatar/avatar.html" with classes="avatar--small" %}
```

Comment:
I like this - let's see how it goes with maintaining it, and if it proves onerous we can abandon it. Could you update the design-guide docs to mention this specifically?

Reply:
Sounds good, let's see how it goes. Aware that maintaining the pattern lib is a task in itself so conscious of not creating more work for ourselves. Added in https://github.com/torchbox/torchbox.com/pull/19/commits/01707efde56ba1a21b73f85b73009b4682ffeb3d